migration/cuju-ft-trans-file.c

```diff
@@ -574,7 +600,13 @@ static int cuju_ft_trans_try_load(CujuQEMUFileFtTrans *s)
 #endif
 
     while (s->ft_serial == ft_serial && cuju_ft_trans_load_ready(s)) {
-        ret = cuju_ft_trans_send_header(s, CUJU_QEMU_VM_TRANSACTION_ACK1, 0);
+        ret = cuju_ft_trans_send_header(s,s->check<<CUJU_FT_ALIVE_HEADER|CUJU_QEMU_VM_TRANSACTION_ACK1, 0);
```

Comment:
Please add some description for this part
(or maybe copy CUJU_FT_ALIVE_HEADER define comment)

Reply:
add some description:
/*
            if backup receive checkalive header then s->check = 1 and the ACK1 header would set leftmost bit to be 1.   
            so s->check<<CUJU_FT_ALIVE_HEADER equal to 1<<15.
*/